WordPress/Classes/Utility/Blogging Reminders/BloggingRemindersScheduleFormatter.swift

```diff
@@ -113,6 +113,13 @@ private extension BloggingRemindersScheduleFormatter {
     }
 
     static func stringToAttributedString(_ string: String) -> NSAttributedString {
+        // When the app is in a background state, return the non-emphasized plain String instead.
+        // Parsing a HTML string in the background may lead to a crash.
+        // Refer to ReaderCommentsViewController.m:1088 for a similar scenario and explanation.
```

Comment:
Referring to a a file and line number may get outdated. If anyone were to update `ReaderCommentsViewController.m`, this could reference the wrong point in the file or even if someone were to remove that comment it could be missing from that file completely. I don't know a great solution to this because anything here could lead to this problem. The only thing I can think of is a link to the file using its latest commit but that wouldn't look all that great since it'd be a long URL: `https://github.com/wordpress-mobile/WordPress-iOS/blob/28ebe98a054d2bf20ddd77dc70bf6390250c94c7/WordPress/Classes/ViewRelated/Reader/Comments/ReaderCommentsViewController.m#L1088-L1097`

Reply:
Makes sense @wargcm. In https://github.com/wordpress-mobile/WordPress-iOS/pull/20284/commits/e18784ed6695d5d072732c6a992ec17cc4bac70b, I've updated this to refer to the PR instead.